src/index.ts

```diff
@@ -172,6 +172,17 @@ const Database = {
         return await collection.find<T>({ [key]: value }).toArray();
     },
 
+    fetchPartialData: async <T>(data: Filter<Object>, collection: string): Promise<T | null> => {
```

Comment:
Add relevant tests for this.

Reply:
`
const account: IAccountModel = await Database.fetchPartialData<IAccountModel>({
          _id: new ObjectId(decoded.id),
          JwtToken: token, 
}, "accounts");
`

The functions I have implemented there I already use actively in my code. 
Currently i cant send you tests but this is one of the code what i use and its working.